src/project/qml/MuseScore/Project/internal/ProjectPropertiesTopPanel.qml

```diff
@@ -0,0 +1,122 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+
+Column {
+    id: root
+
+    Layout.fillWidth: true
+
+    spacing: 4
+
+    property int propertyNameWidth: 110
+    property NavigationPanel navigationPanel: null
+
+    RowLayout {
+        anchors.left: parent.left
+        anchors.right: parent.right
+
+        spacing: 8
+
+        PropertyItem {
+            propertyNameWidth: root.propertyNameWidth
+            propertyItem: {
+                "index": 0,
+                "propertyName": qsTrc("project", "File path:"),
+                "propertyValue": projectPropertiesModel.filePath,
+                "isStandardProperty": true,
+                "isTopPanelProperty": true
```

Comment:
As far I understand, this property should be renamed to "readonly"

Reply:
@RomanPudashkin I am also not changing isStandardProperty. So I am going to mark that as readonly too

Reply:
@RomanPudashkin, when I do that, I get an error: invalid property assignment because isTopPanelProperty is readonly and I'm giving it a value here
![image](https://user-images.githubusercontent.com/88136054/163662389-79421b89-bf37-41aa-a4c4-7b390fa64291.png)